index.html

```diff
@@ -18,8 +20,8 @@
                 <div class="logo">
                     <a href="/">
                         <picture>
-                            <source srcset="/src/header/logo.svg" media="(min-width: 769px)"/>
-                            <source srcset="/src/header/logo_mobile.svg" media="(min-width: 120px)"/>
+                            <source srcset="/src/header/logo.svg" media="(min-width: 769px)">
+                            <source srcset="/src/header/logo_mobile.svg" media="(min-width: 120px)">
```

Comment:
Do we need this `/ `in the path to images?

```suggestion
                            <source srcset="src/header/logo.svg" media="(min-width: 769px)">
                            <source srcset="src/header/logo_mobile.svg" media="(min-width: 120px)">
```

Reply:
fixed